include/graphblas/nonblocking/coordinates.hpp

```diff
@@ -303,6 +303,12 @@ namespace grb {
 					}
 				}
 
+				template< bool maybe_invalid = false >
+				inline void assignAll( ) noexcept {
```

Comment:
no space between `( )`

Reply:
Is the declaration of this new function perhaps not necessary? I don't see where it is called

Reply:
reference/vector.hpp:874 is expecting the internal Coordinates to provide such function